### PR TITLE
Add collapsible to improve readability

### PR DESCRIPTION
**Pull request by EDDYMENS**
Wrapped the dumped structure objects in a collapsible to make ReadMe easier to read.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -439,6 +441,9 @@ class Sabberworm\CSS\CSSList\Document#4 (2) {
 
 ```
 
+</details>
+
+
```
Comment (oliverklee) [inline README.md:446]: For consistency, please let's not have two blank lines in a row in the README.

File: README.md [hunk 2]
```diff
@@ -606,6 +613,8 @@ class Sabberworm\CSS\CSSList\Document#4 (2) {
 }
 
 ```
+</details>
+
```
Comment (oliverklee) [inline README.md:617]: Ditto.

**Review by oliverklee — CHANGES_REQUESTED** [review]
I like the change a lot!

I've left two comments on Markdown formatting nitpicks.

File: README.md (hunk 1 above)
Comment (EDDYMENS) [reply README.md]: The extra newline is gone now :)

**Comment by sabberworm** [comment]
Thanks. I like the increased usability this brings but I worry “Click to expand” is not a title saying much. Would it be possible to use the preceding `h4` as a summary?

**Comment by EDDYMENS** [comment]
@sabberworm good idea! Using the `h4` tag pushes the text to the next line so I used `<b>` instead and also `<code>` for highlighting since the backtick ` won't work in this case as well. 
Let me know if this is ok

**Pull request merged**